src/tlo/methods/labour.py

```diff
@@ -2891,6 +2917,11 @@ def apply(self, person_id, squeeze_factor):
             self.module, self, self.module.item_codes_lab_consumables, core='delivery_core',
             optional='delivery_optional')
 
+        # Log required equipment
+        self.EQUIPMENT.update({'Delivery set', 'Weighing scale', 'Stethoscope, foetal, monaural, Pinard, plastic',
+                               'Resuscitaire', 'Sphygmomanometer', 'Tray, emergency', 'Suction machine',
+                               'Thermometer', 'Drip stand', 'Infusion pump'})
+
```

Comment:
Will the equipment be used if the consumables above are not available?

Reply:
these are general equipment that should be available/used at all births - i think fine to leave as is now.